documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -6,16 +6,28 @@ Hot Rod is a binary TCP protocol that {brandname} provides for high-performance
 
 .Client intelligence
 
-The Hot Rod protocol includes a mechanism that provides clients with an up to date view of the cache topology.
+The Hot Rod protocol includes a mechanism that provides clients with an up-to-date view of the cache topology.
 Client intelligence improves performance by reducing the number of network hops for read and write operations.
 
+`HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism. `HASH_DISTRIBUTION_AWARE` is the best mechanism for allowing clients to route requests to primary owners, which improves a client's performance.
```

Comment:
```suggestion
`HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism and allows clients to route requests to primary owners, providing the best performance for Hot Rod clients.
```

Reply:
Thanks, @oraNod . I implemented your suggestion with some slight tweaks. I usually avoid the verb "allow" in my content, but it's not anthropomorphic in this instance, so all G!